app/src/main/java/com/example/newsapp/ui/home/viewmodels/ChangePasswordViewModel.kt

```diff
@@ -33,19 +33,17 @@ class ChangePasswordViewModel @Inject constructor(
 
     var errorMessage = MutableLiveData<String>()
 
-
     fun checkUserPassword(oldPassword: String, newPassword: String) {
-
         viewModelScope.launch {
             val user = repository.getUser(userEmail)
             if (user != null) {
```

Comment:
@jnandrawos no need for such null checks in Kotlin, you can use safe calls like ?.let { }